doc/classes/EditorSettings.xml

```diff
@@ -1133,6 +1133,24 @@
 			A touch-friendly panel that provides easy access to common actions such as save, delete, undo, and redo without requiring a keyboard.
 			[b]Note:[/b] Only available in the Android and XR editor.
 		</member>
+		<member name="naming/default_signal_callback_name" type="String" setter="" getter="">
+			The format of the default signal callback name (in the Signal Connection Dialog). The following substitutions are available: [code]{NodeName}[/code], [code]{nodeName}[/code], [code]{node_name}[/code], [code]{SignalName}[/code], [code]{signalName}[/code], and [code]{signal_name}[/code].
+		</member>
+		<member name="naming/default_signal_callback_to_self_name" type="String" setter="" getter="">
+			The format of the default signal callback name when a signal connects to the same node that emits it (in the Signal Connection Dialog). The following substitutions are available: [code]{NodeName}[/code], [code]{nodeName}[/code], [code]{node_name}[/code], [code]{SignalName}[/code], [code]{signalName}[/code], and [code]{signal_name}[/code].
+		</member>
+		<member name="naming/node_name_casing" type="int" setter="" getter="">
+			When creating node names automatically, set the type of casing to use in this project. This is mostly an editor setting.
```

Comment:
We can nicely get rid of the last sentence now. Plus "Set". As if this were a method but it's a _setting_.
```suggestion
			When creating node names automatically, the type of casing to use in this project.
```